### PR TITLE
[wg/fedid] Horizontal Review

### PR DESCRIPTION
**Pull request by simoneonofri**
Changes
 - specified privacy properties for Decentralize Identidy Model

### CODE REVIEW

File: 2024/wg-fedid.html [hunk 1]
```diff
@@ -313,11 +313,13 @@ <h2>Success Criteria</h2>
 
       <!-- Horizontal review -->
 
-      <p>Each specification should contain a Security Considerations section that must include a Threat Model with threats, attacks, mitigations, and residual risks and a Privacy Consideration section as specified in <a href="https://www.w3.org/TR/security-privacy-questionnaire/">Self-Review Questionnaire: Security and Privacy</a> and <a href="https://datatracker.ietf.org/doc/html/rfc3552">RFC 3552</a>, detailing all known security and privacy implications for implementers, Web authors, and end users.</p>
+      <p>
+        Each specification should contain a Security Considerations section -  that must include a Threat Model with threats, attacks, mitigations, and residual risks - and a Privacy Consideration section - that must contain an analysis of privacy aspects such as Unlinkability, Data Minimization and Tracking -  as specified in <a href="https://www.w3.org/TR/security-privacy-questionnaire/">Self-Review Questionnaire: Security and Privacy</a> and <a href="https://datatracker.ietf.org/doc/html/rfc3552">RFC 3552</a>, detailing all known security and privacy implications for implementers, Web authors, and end users.
```
Comment (msporny) [inline 2024/wg-fedid.html]:
```suggestion
        Each specification will contain a Security Considerations section -  that includes a Threat Model with threats, attacks, mitigations, and residual risks - and a Privacy Consideration section - that must contain an analysis of privacy aspects such as Unlinkability, Data Minimization and Tracking -  as specified in <a href="https://www.w3.org/TR/security-privacy-questionnaire/">Self-Review Questionnaire: Security and Privacy</a> and <a href="https://datatracker.ietf.org/doc/html/rfc3552">RFC 3552</a>, detailing all known security and privacy implications for implementers, Web authors, and end users.
```

**Review by msporny — APPROVED** [review]

File: 2024/wg-fedid.html (hunk 1 above)
Comment (aniltj) [reply 2024/wg-fedid.html]: Agree that with suggested change by @msporny that the security considerations and the privacy considerations sections are a SHALL (a requirement) and not just a SHOULD (a recommendation)

**Review by samuelgoto — APPROVED** [review]
This seems like a reasonable and constructive addition to the charter. LGTM++.

**Comment by hlflanagan** [comment]
+1

**Pull request merged**